plugins/modules/source_control/gitlab/gitlab_branch.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Werner Dijkerman (ikben@werner-dijkerman.nl)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+module: gitlab_branch
+short_description: Create/delete branch
+description:
+  - Create/Delete branches
+author:
+  - Werner Dijkerman (@dj-wasabi)
```

Comment:
```suggestion
  - Werner Dijkerman (@dj-wasabi)
  - You have to add yourself here
```

Reply:
tests added.

Reply:
You still need to add yourself here.

Reply:
I can't, i'm not in maintainers list. I have an error with checks when I do it.

Reply:
You need to add yourself to the maintainers list for this module as well.

Reply:
How do that?

Reply:
Simply add a new entry like https://github.com/ansible-collections/community.general/blob/main/.github/BOTMETA.yml#L950-L951 for the new module.